src/completers/docker.zsh

```diff
@@ -6,7 +6,7 @@ colors
 _fzf_complete_docker() {
     local subcommand=${${(Q)${(z)@}}[2]}
 
-    if [[ $subcommand =~ ^(create|history|run)$ ]]; then
+    if [[ $subcommand =~ ^(create|history|push|run)$ ]]; then
```

Comment:
As a design, it is confusing if the result of completion is the 2nd column when it comes after `docker push **<TAB>` but is the 1st column otherwise. It should rather show the different table to make this straightforward.